README.md

```diff
@@ -1,32 +1,50 @@
 Syslog Protocol DFDL Schema
 ====
-Support for:
-- RFC-5424
-- RFC-3164
-- Cisco
-- ESXi
+Includes support for the following syslog variants:
 
-There are two root elements to choose from:
+- RFC-5424 (`syslog-rfc5424.dfdl.xsd`)
+- RFC-3164 (`syslog-rfc3164.dfdl.xsd`)
+- Cisco (`syslog-cisco.dfdl.xsd`)
+- ESXi (`syslog-esxi.dfdl.xsd`)
 
-Syslog - straight RFC 5423 logging
+Each file defines the format of a specific variant as well as a root element
+that can be used to parse only that variant.
 
-SolarwindsSyslog - The combined Solarwinds Syslog
-format which supports all the above-listed formats. 
+To support all possible variants, the `syslog.dfdl.xsd` file defines a `Syslog`
+root element.
 
-This schema works with the Daffodil DFDL implementation.
-It uses dfdl:inputValueCalc and dfdl:outputValueCalc in 
-the Facility and Severity elements. 
+To support all possible variants from a Solarwinds log forwarder, which inserts
+data immediately after the facility/severity information, the
+`syslog-solarwinds.dfdl.xsd` file defines a `SolarwindsSyslog` root element.
 
-A portable version works also with IBM DFDL.
-This doesn't parse the Prival information into
-Facility and Severity, but is otherwise identical.   
+## Portability
 
-Change History
+This schema is known to work with the Daffodil DFDL implementation. It uses
+dfdl:inputValueCalc and dfdl:outputValueCalc in the Facility and Severity
+elements, which do not work with IBM DFDL.
 
-* 0.1.0: Initial support for RFC-5424, RFC-3164 and Cisco
-* 0.1.1: Support for ESXi
-* 0.1.2: Support for Solarwinds as well as standard Syslog in same schema.
-* 1.0.0: Portable variant that works with IBM DFDL.
-Removed version number from namespace URI. 
-Changed branding from Tresys to Owl. 
+The common.dfdl.xsd file can be modified to enable support for IBM DFDL. See
+the documentation annotations in the Prival groups for details on the needed
+adjustmens. Additionally, encodingErrorPolicy must be changed from "replace" to
+"error" when used with IBM DFDL.
 
+## Change History
```

Comment:
I like to call this section Release Notes because we're not trying to recapture everything that is in the git commit log comments, just what is important about the release.

Reply:
Makes sense, I'll update and make sure my additions focus on the more important aspects.